parm/config/config.gldas

```diff
@@ -12,9 +12,7 @@ export GLDASSH=$HOMEgfs/scripts/exgdas_gldas.sh.ecf
 export gldas_spinup_hours=72
 
 
-if [ $machine = "WCOSS_C" -o $machine = "WCOSS_DELL_P3" ]; then
-    export CPCGAUGE=/gpfs/dell2/emc/modeling/noscrub/Fanglin.Yang/stat/cpcUniGauge
```

Comment:
Is this supposed to be removed for Cray/Dell?

Reply:
already removed see the right hand column